packages/main/src/MenuItem.hbs

```diff
@@ -0,0 +1,113 @@
+{{>include "./ListItem.hbs"}}
+
+{{#if hasSubmenu}}
+	{{> listItemPostContent}}
+{{/if}}
+
+{{#*inline "listItemContent"}}
+	{{#if text}}
+		<div class="ui5-menu-item-text">{{text}}</div>
+	{{/if}}
+	{{#if additionalText}}
```

Comment:
Is it possible to optimize output to use same container for submenu arrow (>) and shortcut text as the both appear at the end of the item, right aligned, etc? Later when we add additional actions, they also will appear there in the same way. The priority is [will be] Submenu arrow - [actions] - shortcut text.

Reply:
Done